SPECS/libXau/libXau.signatures.json

```
@@ -1,5 +1,5 @@
 {
  "Signatures": {
-  "libXau-1.0.9.tar.bz2": "ccf8cbf0dbf676faa2ea0a6d64bcc3b6746064722b606c8c52917ed00dcb73ec"
+  "libXau-1.0.11.tar.gz": "3a321aaceb803577a4776a5efe78836eb095a9e44bbc7a465d29463e1a14f189"
  }
 }
```

nit - end of file newline
```suggestion
}

```